account_bank_statement_import_paypal/wizards/account_bank_statement_import_paypal.py

```diff
@@ -66,7 +66,11 @@ def _paypal_convert_amount(self, amount_str):
 
     @api.model
     def _check_paypal(self, data_file):
-        data_file = self._get_paypal_str_data(data_file)
+        try:
+            data_file = self._get_paypal_str_data(data_file)
+        except UnicodeDecodeError as e:
+            _logger.debug(e)
+            return False
         if not self.paypal_map_id:
```

Comment:
I'm thinking that it's better also to put this at the beginning. Would you mind?